tests/test_embed/test_interpreter.cpp

```diff
@@ -323,3 +323,32 @@ TEST_CASE("sys.argv gets initialized properly") {
     }
     py::initialize_interpreter();
 }
+
+class Counter {
+public:
+    Counter() { cnt_++; }
+    Counter(const Counter &) { cnt_++; }
+    Counter(Counter &&) { cnt_++; }
```

Comment:
```suggestion
    Counter(Counter &&) noexcept { cnt_++; }

Reply:
Thanks for your suggestions. Fixed in newer commits.